geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionStats.java

```diff
@@ -90,7 +91,8 @@
   private static final int functionExecutionQueueThrottleCountId;
   private static final int functionExecutionQueueThrottleTimeId;
   private static final int serialQueueSizeId;
-  private static final int serialQueueBytesId;
+  @VisibleForTesting
```

Comment:
Rather than making the member visible, can we make a member function visible instead. If all that is needed is read access to the current value this should help prevent any future abuse of using this loosened access for updates.

Reply:
good catch, but in this case serialQueueBytesId is static, so it can only be updated once.